packages/flutter/lib/src/material/scaffold.dart

```diff
@@ -978,6 +1289,9 @@ class ScaffoldState extends State<Scaffold> with TickerProviderStateMixin {
       bottomSheet.animationController.dispose();
     if (_currentBottomSheet != null)
       _currentBottomSheet._widget.animationController.dispose();
+    _fabMoveController.removeStatusListener(_onFabMoveStatus);
+    _fabMoveController.dispose();
+    _fabMoveController = null;
```

Comment:
no need to mark it as null, the class is defunct after this anyway

Reply:
Done.

Currently, the SnackBarController is also getting set to null.  It's also an animation controller, so I have gone ahead and removed the line that was nulling it out to be consistent here.